database/functions/author/create.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE FUNCTION CREATE_AUTHOR(
+    country_id INT,
+    firstname CHAR(50),
+    lastname CHAR(50)
+) RETURNS INT
+LANGUAGE plpgsql    
+AS $$
+DECLARE res INT;
+BEGIN
+    INSERT INTO AUTHOR(COUNTRY_ID, FIRSTNAME, LASTNAME)
+    VALUES(country_id, firstname, lastname)
+    RETURNING ID INTO res;
```

Comment:
I dont's thin that use variable of res is needed here, you can just use RETURNING ID

Reply:
can't come up with the working solution without a variable :(